docs/API.md

```diff
@@ -773,3 +773,29 @@ $ curl -X POST -H "Content-Type: application/json" -d '{"debian-7-i386":"templat
   "ok": true
 }
 ```
+
+##### POST /poolreset
+
+Clear all pending and ready instances in a pool, and deploy replacements
+
+All pool reset requests must be for pools that exist in the vmpooler configuration running, or a 404 code will be returned.
+
+When a pool reset is requested a 201 status will be returned.
+
+A pool reset will cause vmpooler manager to log that it has cleared ready and pending instances.
+
+For poolreset to be available it is necessary to enable experimental features. Additionally, the request must be performed with an authentication token when authentication is configured.
+
+Responses:
+* 201 - Pool reset requested received
+* 400 - An invalid configuration was provided causing requested changes to fail
+* 404 - An unknown error occurred
+* 405 - The endpoint is disabled because experimental features are disabled
+```
+$ curl -X POST -H "Content-Type: application/json" -d '{"debian-7-i386":"1"}' --url https://vmpooler.example.com/api/v1/poolreset
```

Comment:
I see that the number (poolcount) is not used, but is it needed to pass the validation? would a JSON like `{"debian-7-i386"}` also work?

Reply:
@sbeaulie, I would have preferred to skip the number, but yes, it being json made me thing that is necessary. I did spend some time attempting to replicate the behavior of the vm endpoint where we can pass a pool title as a part of the URL, instead of having to pass in a data block. However, I was not able to make this work, so eventually accepted having the number so I could push this over the line.